README.rst

```diff
@@ -148,12 +148,14 @@ You can run it with Docker as well
   # Download the image
   $ docker pull cloudcustodian/c7n
 
-  # Run the policy
+  # Run the policy (AWS)
   $ docker run -it \
-      -v $(pwd)/output:/output \
-      -v $(pwd)/policy.yml:/policy.yml \
-      --env-file <(env | grep "^AWS") \
-      cloudcustodian/c7n run -v -s /output /policy.yml
+    -v $(pwd)/output:/output \
+    -v $(pwd)/policy.yml:/policy.yml \
+    -v $(cd ~ | pwd)/.aws/credentials:/root/.aws/credentials \
```

Comment:
feels like most of this is about to break due to root paths on the right hand side

while in some cases credentials file is appropriate and needed it also feels potentially like we should talk about why this line is here, because its actually providing potentially alot of credentials access (ie non minimal) which would be a turn off to those who are extra security conscious.